### PR TITLE
Samples update 5.2

### PR DESCRIPTION
**Pull request by danyialkhan**

### CODE REVIEW

File: dotnetcore/BackingSource/BackingSourceProvider.Sql.Netcore/BackingSourceProvider.Sql.Netcore.csproj [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<Project Sdk="Microsoft.NET.Sdk">
```
Comment (Sarah-Hameed) [inline dotnetcore/BackingSource/BackingSourceProvider.Sql.Netcore/BackingSourceProvider.Sql.Netcore.csproj:1]: Why this file is added?

File: dotnetcore/BackingSource/BackingSourceProvider.Sql.Netcore/Class1.cs [hunk 2]
```diff
@@ -0,0 +1,8 @@
+﻿using System;
+
```
Comment (Sarah-Hameed) [inline dotnetcore/BackingSource/BackingSourceProvider.Sql.Netcore/Class1.cs:2]: Why is this file added

Comment (danyialkhan) [reply dotnetcore/BackingSource/BackingSourceProvider.Sql.Netcore/Class1.cs]: removed this unused file

File: dotnetcore/DataStructures/CacheAttributes/README.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+For this sample, Distributed List is used for demonstration. Other data structures can be used in a similar way.
+
+### Prerequisites
+
+Before the sample application is executed make sure that:
+
+- app.config have been changed according to the configurations. 
+	- Change the cache name
+- By default this sample uses 'myPartitionedCache', make sure that cache is running.
```
Comment (Sarah-Hameed) [inline dotnetcore/DataStructures/CacheAttributes/README.md:25]: "democache"

File: dotnetcore/CustomEvents/README.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+	clustering feature to communicate with other users. When you enter your message, a custom notification about the new message is received at all other nodes in the cluster.
+	
+### Prerequisites
+
+Before the sample application is executed make sure that:
+
+- app.config have been changed according to the configurations. 
+	- Change the cache name 
+	
+- By default this sample uses 'myPartitionedCache', make sure that cache is running.
```
Comment (Sarah-Hameed) [inline dotnetcore/CustomEvents/README.md:25]: "democache"

File: dotnetcore/BulkCustomDependency/README.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+- app.config have been changed according to the configurations. 
+	- Change the cache name
+	- connectionString:	the connection string used to connect to the database. By default it connects to the localhost.
+- To deploy Custom Dependency, Build 'CustomDependencyImpl' project.
+	- Start NCache Web Manager and create a clustered cache with the name specified in app.config.
+	- Click on 'Deploy Providers' in the Cache Setting(...) of clustered cache's page.
+	- Select CustomDependencyImpl.dll and click Ok.
+	- Save changes.
+
+- By default this sample uses 'myPartitionedCache', make sure that cache is running.
```
Comment (Sarah-Hameed) [inline dotnetcore/BulkCustomDependency/README.md:32]: "democache"

File: dotnetcore/BulkCustomDependency/BulkCustomDependencyUsage/App.config [hunk 6]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <appSettings>
+    <add key="CacheID" value="demoCache"/>
+    <add key="connectionString" value="Data Source=20.200.20.10;Database=Northwind-QA;User ID=diyatech;password=4Islamabad;"/>
```
Comment (Sarah-Hameed) [inline dotnetcore/BulkCustomDependency/BulkCustomDependencyUsage/App.config:5]: remove these credentials

File: dotnetcore/DataStructures/DistributedDictionary/README.md [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+This sample program demonstrates how to use the NCache Distributed Dictionary API. 
+It shows how to initialize the cache instance and perform Dictionary Data Structure related tasks from NCache.
+
+### Prerequisites
+
+Before the sample application is executed make sure that:
+
+- app.config have been changed according to the configurations. 
+	- Change the cache name
+- By default this sample uses 'myPartitionedCache', make sure that cache is running.
```
Comment (Sarah-Hameed) [inline dotnetcore/DataStructures/DistributedDictionary/README.md:23]: "democache"

File: dotnetcore/DataStructures/DistributedHashSet/README.md [hunk 8]
```diff
@@ -0,0 +1,48 @@
...
+This sample program demonstrates how to use the NCache Distributed HashSet API. 
+It shows how to initialize the cache instance and perform HashSet Data Structure related tasks from NCache.
+
+### Prerequisites
+
+Before the sample application is executed make sure that:
+
+- app.config have been changed according to the configurations. 
+	- Change the cache name
+- By default this sample uses 'myPartitionedCache', make sure that cache is running.
```
Comment (Sarah-Hameed) [inline dotnetcore/DataStructures/DistributedHashSet/README.md:23]: democache

File: dotnetcore/DataStructures/DistributedQueue/README.md [hunk 9]
```diff
@@ -0,0 +1,50 @@
...
+
+This sample uses SampleData project as a reference for model class "Customer".
+
+### Prerequisites
+
+Before the sample application is executed make sure that:
+
+- app.config have been changed according to the configurations. 
+	- Change the cache name
+- By default this sample uses 'myPartitionedCache', make sure that cache is running.
```
Comment (Sarah-Hameed) [inline dotnetcore/DataStructures/DistributedQueue/README.md:25]: democache

File: dotnetcore/Dependencies/OracleDependency/README.md [hunk 10]
```diff
@@ -0,0 +1,56 @@
...
+
+- Change notification privileges must be granted.
+```
+grant change notification to [username]
+```
+- app.config have been changed according to the configurations. 
+	- Change the cache name 
+	- conn-string to connect with database.
+	
+- By default this sample uses 'myPartitionedCache', make sure that cache is running.
```
Comment (Sarah-Hameed) [inline dotnetcore/Dependencies/OracleDependency/README.md:31]: democache

File: dotnet/BasicOperationsWithJSON/README.md [hunk 11]
```diff
@@ -47,4 +47,4 @@ Alachisoft [C] provides various sources of technical support.
 
 ### Copyrights
 
-[C] Copyright 2019 Alachisoft
```
Comment (Sarah-Hameed) [inline dotnet/BasicOperationsWithJSON/README.md:50]: democache

File: dotnet/DataStructures/CacheAttributes/CacheAttributes/Properties/AssemblyInfo.cs [hunk 12]
```diff
@@ -0,0 +1,36 @@
...
+
+// General Information about an assembly is controlled through the following
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyTitle("DTCacheAttributes")]
+[assembly: AssemblyDescription("")]
+[assembly: AssemblyConfiguration("")]
+[assembly: AssemblyCompany("")]
+[assembly: AssemblyProduct("DTCacheAttributes")]
+[assembly: AssemblyCopyright("Copyright ©  2018")]
```
Comment (Sarah-Hameed) [inline dotnet/DataStructures/CacheAttributes/CacheAttributes/Properties/AssemblyInfo.cs:13]: update copyrights

File: dotnet/DataStructures/CacheAttributes/README.md [hunk 13]
```diff
@@ -0,0 +1,48 @@
...
+This sample program demonstrates how to use the NCache Distributed Data Structure with Cache attributes. 
+It shows how to initialize the cache instance and perform  Data Structure related tasks like Locking/Unlocking, Register/Unregister for Notification fromm NCache.
+
+### Prerequisites
+
+Before the sample application is executed make sure that:
+
+- app.config have been changed according to the configurations. 
+	- Change the cache name
+- By default this sample uses 'myPartitionedCache', make sure that cache is running.
```
Comment (Sarah-Hameed) [inline dotnet/DataStructures/CacheAttributes/README.md]: democache

File: dotnet/DataStructures/DistributedDictionary/DistributedDictionary/DistributedDictionary.csproj [hunk 14]
```diff
@@ -0,0 +1,134 @@
...
+    <PlatformTarget>AnyCPU</PlatformTarget>
+    <DebugType>pdbonly</DebugType>
+    <Optimize>true</Optimize>
+    <OutputPath>bin\Release\</OutputPath>
+    <DefineConstants>TRACE</DefineConstants>
+    <ErrorReport>prompt</ErrorReport>
+    <WarningLevel>4</WarningLevel>
+  </PropertyGroup>
+  <ItemGroup>
+    <Reference Include="Alachisoft.NCache.Cache, Version=5.2.0.0, Culture=neutral, PublicKeyToken=cff5926ed6a53769, processorArchitecture=MSIL">
```
Comment (Sarah-Hameed) [inline dotnet/DataStructures/DistributedDictionary/DistributedDictionary/DistributedDictionary.csproj:36]: fix the reference

File: dotnet/DataStructures/DistributedCounter/DistributedCounter/DistributedCounter.csproj [hunk 15]
```diff
@@ -0,0 +1,136 @@
...
+    <PlatformTarget>AnyCPU</PlatformTarget>
+    <DebugType>pdbonly</DebugType>
+    <Optimize>true</Optimize>
+    <OutputPath>bin\Release\</OutputPath>
+    <DefineConstants>TRACE</DefineConstants>
+    <ErrorReport>prompt</ErrorReport>
+    <WarningLevel>4</WarningLevel>
+  </PropertyGroup>
+  <ItemGroup>
+    <Reference Include="Alachisoft.NCache.Cache, Version=5.2.0.0, Culture=neutral, PublicKeyToken=cff5926ed6a53769, processorArchitecture=MSIL">
```
Comment (Sarah-Hameed) [inline dotnet/DataStructures/DistributedCounter/DistributedCounter/DistributedCounter.csproj:35]: fix the reference

File: dotnet/DataStructures/CacheAttributes/CacheAttributes/CacheAttributes.csproj [hunk 16]
```diff
@@ -0,0 +1,135 @@
...
+    <DebugType>pdbonly</DebugType>
+    <Optimize>true</Optimize>
+    <OutputPath>bin\Release\</OutputPath>
+    <DefineConstants>TRACE</DefineConstants>
+    <ErrorReport>prompt</ErrorReport>
+    <WarningLevel>4</WarningLevel>
+  </PropertyGroup>
+  <ItemGroup>
+    <Reference Include="Alachisoft.NCache.Cache, Version=5.2.0.0, Culture=neutral, PublicKeyToken=cff5926ed6a53769, processorArchitecture=MSIL">
+      <HintPath>..\packages\Alachisoft.NCache.SDK.5.2.0\lib\net40\Alachisoft.NCache.Cache.dll</HintPath>
```
Comment (Sarah-Hameed) [inline dotnet/DataStructures/CacheAttributes/CacheAttributes/CacheAttributes.csproj:37]: fix package reference

File: dotnet/DataStructures/DistributedCounter/README.md [hunk 17]
```diff
@@ -0,0 +1,48 @@
...
+This sample program demonstrates how to use the NCache Distributed Counter API. 
+It shows how to initialize the cache instance and perform Counter Data Structure related tasks from NCache.
+
+### Prerequisites
+
+Before the sample application is executed make sure that:
+
+- app.config have been changed according to the configurations. 
+	- Change the cache name
+- By default this sample uses 'myPartitionedCache', make sure that cache is running.
```
Comment (Sarah-Hameed) [inline dotnet/DataStructures/DistributedCounter/README.md]: democache

File: dotnet/DataStructures/DistributedDictionary/README.md [hunk 18]
```diff
@@ -0,0 +1,48 @@
...
+This sample program demonstrates how to use the NCache Distributed Dictionary API. 
+It shows how to initialize the cache instance and perform Dictionary Data Structure related tasks from NCache.
+
+### Prerequisites
+
+Before the sample application is executed make sure that:
+
+- app.config have been changed according to the configurations. 
+	- Change the cache name
+- By default this sample uses 'myPartitionedCache', make sure that cache is running.
```
Comment (Sarah-Hameed) [inline dotnet/DataStructures/DistributedDictionary/README.md]: democache

File: dotnet/DataStructures/DistributedHashSet/DistributedHashSet/DistributedHashSet.csproj [hunk 19]
```diff
@@ -0,0 +1,136 @@
...
+    <PlatformTarget>AnyCPU</PlatformTarget>
+    <DebugType>pdbonly</DebugType>
+    <Optimize>true</Optimize>
+    <OutputPath>bin\Release\</OutputPath>
+    <DefineConstants>TRACE</DefineConstants>
+    <ErrorReport>prompt</ErrorReport>
+    <WarningLevel>4</WarningLevel>
+  </PropertyGroup>
+  <ItemGroup>
+    <Reference Include="Alachisoft.NCache.Cache, Version=5.2.0.0, Culture=neutral, PublicKeyToken=cff5926ed6a53769, processorArchitecture=MSIL">
```
Comment (Sarah-Hameed) [inline dotnet/DataStructures/DistributedHashSet/DistributedHashSet/DistributedHashSet.csproj:35]: fix the reference

File: dotnet/DataStructures/DistributedQueue/README.md [hunk 20]
```diff
@@ -0,0 +1,50 @@
...
+
+This sample uses SampleData project as a reference for model class "Customer".
+
+### Prerequisites
+
+Before the sample application is executed make sure that:
+
+- app.config have been changed according to the configurations. 
+	- Change the cache name
+- By default this sample uses 'myPartitionedCache', make sure that cache is running.
```
Comment (Sarah-Hameed) [inline dotnet/DataStructures/DistributedQueue/README.md]: democache

File: dotnet/DataStructures/DistributedDictionary/DistributedDictionary/DistributedDictionary.csproj (hunk 14 above)
Comment (danyialkhan) [reply dotnet/DataStructures/DistributedDictionary/DistributedDictionary/DistributedDictionary.csproj:36]: It is expected behaviour in the case of the DotNet framework, it adds a reference from the packages folder.

File: dotnet/Dependencies/CustomDependencyBulk/CustomDependencyBulkUsage/App.config [hunk 21]
```diff
@@ -5,8 +5,9 @@
       type="OracleInternal.Common.ODPMSectionHandler, Oracle.ManagedDataAccess, Version=4.122.19.1, Culture=neutral, PublicKeyToken=89b483f429c47342"/>
     </configSections>
   <appSettings>
-    <add key="CacheID" value="myPartitionedCache"/>
-    <add key="connectionString" value="Data Source=localhost;Database=Northwind;User ID=sa;password=password;"/>
+    <add key="CacheID" value="demoCache"/>
+    <add key="ConnectionString" value="Data Source=localhost;Database=Northwind;User ID=sa;password=password;"/>
```
Comment (Sarah-Hameed) [inline dotnet/Dependencies/CustomDependencyBulk/CustomDependencyBulkUsage/App.config:9]: fix connection string

File: dotnet/Dependencies/OracleDependency/OracleDependency/OracleDependency.csproj [hunk 22]
```diff
@@ -83,57 +68,15 @@
...
-    </Reference>
-    <Reference Include="Alachisoft.NCache.Runtime, Version=5.0.0.0, Culture=neutral, PublicKeyToken=cff5926ed6a53769, processorArchitecture=MSIL">
-      <HintPath>..\packages\Alachisoft.NCache.SDK.5.0.2\lib\net40\Alachisoft.NCache.Runtime.dll</HintPath>
-    </Reference>
-    <Reference Include="Alachisoft.NCache.Security, Version=5.0.0.0, Culture=neutral, PublicKeyToken=cff5926ed6a53769, processorArchitecture=MSIL">
-      <HintPath>..\packages\Alachisoft.NCache.SDK.5.0.2\lib\net40\Alachisoft.NCache.Security.dll</HintPath>
-    </Reference>
-    <Reference Include="Alachisoft.NCache.Serialization, Version=5.0.0.0, Culture=neutral, PublicKeyToken=cff5926ed6a53769, processorArchitecture=MSIL">
-      <HintPath>..\packages\Alachisoft.NCache.SDK.5.0.2\lib\net40\Alachisoft.NCache.Serialization.dll</HintPath>
+    <Reference Include="Alachisoft.NCache.Client">
```
Comment (Sarah-Hameed) [inline dotnet/Dependencies/OracleDependency/OracleDependency/OracleDependency.csproj:71]: fix this , use SDK

File: dotnet/Dependencies/PollingBasedDependency/PollingDependency/PollingDependency.csproj [hunk 23]
```diff
@@ -63,57 +63,15 @@
     <CodeAnalysisRuleDirectories>;C:\Program Files (x86)\Microsoft Visual Studio 10.0\Team Tools\Static Analysis Tools\FxCop\\Rules</CodeAnalysisRuleDirectories>
   </PropertyGroup>
   <ItemGroup>
-    <Reference Include="Alachisoft.NCache.Cache, Version=5.0.0.0, Culture=neutral, PublicKeyToken=cff5926ed6a53769, processorArchitecture=MSIL">
-      <HintPath>..\packages\Alachisoft.NCache.SDK.5.0.2\lib\net40\Alachisoft.NCache.Cache.dll</HintPath>
+    <Reference Include="Alachisoft.NCache.Client">
+      <HintPath>..\..\..\..\..\bin\assembly\4.0\Alachisoft.NCache.Client.dll</HintPath>
```
Comment (Sarah-Hameed) [inline dotnet/Dependencies/PollingBasedDependency/PollingDependency/PollingDependency.csproj:67]: add proper reference

File: dotnet/Dependencies/PollingBasedDependency/PollingDependency/PollingDependency.csproj [hunk 24]
```diff
@@ -63,57 +63,15 @@
...
-    </Reference>
-    <Reference Include="log4net, Version=1.2.10.0, Culture=neutral, PublicKeyToken=1b44e1d426115821, processorArchitecture=MSIL">
-      <HintPath>..\packages\Alachisoft.NCache.SDK.5.0.2\lib\net40\log4net.dll</HintPath>
-    </Reference>
-    <Reference Include="Oracle.ManagedDataAccess, Version=4.122.19.1, Culture=neutral, PublicKeyToken=89b483f429c47342, processorArchitecture=MSIL">
-      <HintPath>..\packages\Oracle.ManagedDataAccess.19.5.0\lib\net40\Oracle.ManagedDataAccess.dll</HintPath>
-    </Reference>
-    <Reference Include="protobuf-net, Version=2.3.5.0, Culture=neutral, PublicKeyToken=257b51d87d2e4d67, processorArchitecture=MSIL">
-      <HintPath>..\packages\Alachisoft.NCache.SDK.5.0.2\lib\net40\protobuf-net.dll</HintPath>
+    <Reference Include="Alachisoft.NCache.Runtime">
```
Comment (Sarah-Hameed) [inline dotnet/Dependencies/PollingBasedDependency/PollingDependency/PollingDependency.csproj:69]: fix this reference

File: dotnet/Dependencies/SQLDependency/SQLDependency/SQLDependency.csproj [hunk 25]
```diff
@@ -32,57 +32,15 @@
     <WarningLevel>4</WarningLevel>
   </PropertyGroup>
   <ItemGroup>
-    <Reference Include="Alachisoft.NCache.Cache, Version=5.0.0.0, Culture=neutral, PublicKeyToken=cff5926ed6a53769, processorArchitecture=MSIL">
-      <HintPath>..\packages\Alachisoft.NCache.SDK.5.0.2\lib\net40\Alachisoft.NCache.Cache.dll</HintPath>
+    <Reference Include="Alachisoft.NCache.Client">
+      <HintPath>..\..\..\..\..\bin\assembly\4.0\Alachisoft.NCache.Client.dll</HintPath>
```
Comment (Sarah-Hameed) [inline dotnet/Dependencies/SQLDependency/SQLDependency/SQLDependency.csproj:36]: add proper reference

File: dotnet/Dependencies/SQLDependency/SQLDependency/SQLDependency.csproj [hunk 26]
```diff
@@ -32,57 +32,15 @@
...
-    </Reference>
-    <Reference Include="log4net, Version=1.2.10.0, Culture=neutral, PublicKeyToken=1b44e1d426115821, processorArchitecture=MSIL">
-      <HintPath>..\packages\Alachisoft.NCache.SDK.5.0.2\lib\net40\log4net.dll</HintPath>
-    </Reference>
-    <Reference Include="Oracle.ManagedDataAccess, Version=4.122.19.1, Culture=neutral, PublicKeyToken=89b483f429c47342, processorArchitecture=MSIL">
-      <HintPath>..\packages\Oracle.ManagedDataAccess.19.5.0\lib\net40\Oracle.ManagedDataAccess.dll</HintPath>
-    </Reference>
-    <Reference Include="protobuf-net, Version=2.3.5.0, Culture=neutral, PublicKeyToken=257b51d87d2e4d67, processorArchitecture=MSIL">
-      <HintPath>..\packages\Alachisoft.NCache.SDK.5.0.2\lib\net40\protobuf-net.dll</HintPath>
+    <Reference Include="Alachisoft.NCache.Runtime">
```
Comment (Sarah-Hameed) [inline dotnet/Dependencies/SQLDependency/SQLDependency/SQLDependency.csproj:38]: add proper reference

File: dotnet/DistributedLucene/DistributedLucene/README.md [hunk 27]
```diff
@@ -0,0 +1,61 @@
...
+
+### Prerequisites
+
+Before the sample application is executed make sure that:
+
+- A Partitioned OR Partitioned-of-Replica cache is created.
+- app.config have been changed according to the configurations. 
+	- Change the cache name as required.
+	- Change the index name as required.
+- By default this sample uses 'myPartitionedCache' and 'MyIndex', make sure that cache is running.
```
Comment (Sarah-Hameed) [inline dotnet/DistributedLucene/DistributedLucene/README.md]: democache

File: dotnet/EntityFramework/cacheprovider/EFCachingProvider/EFCachingProvider.csproj [hunk 28]
```diff
@@ -0,0 +1,286 @@
...
+    <DefineConstants>TRACE</DefineConstants>
+    <ErrorReport>prompt</ErrorReport>
+    <WarningLevel>4</WarningLevel>
+    <CodeAnalysisRuleSet>AllRules.ruleset</CodeAnalysisRuleSet>
+    <Prefer32Bit>false</Prefer32Bit>
+  </PropertyGroup>
+  <ItemGroup>
+    <Reference Include="Alachisoft.Integrations.EntityFramework.CachingProvider, Version=5.2.0.0, Culture=neutral, PublicKeyToken=cff5926ed6a53769, processorArchitecture=MSIL">
+      <SpecificVersion>False</SpecificVersion>
+      <HintPath>..\..\..\..\..\..\integrations\MSEntityFramework\6.2\Alachisoft.Integrations.EntityFramework.CachingProvider.dll</HintPath>
```
Comment (Sarah-Hameed) [inline dotnet/EntityFramework/cacheprovider/EFCachingProvider/EFCachingProvider.csproj:69]: use proper reference , nuget package

File: dotnet/EntityFramework/objectcaching/EFObjectCaching/EFObjectCaching.csproj [hunk 29]
```diff
@@ -0,0 +1,297 @@
...
+    <OutputPath>bin\Release\</OutputPath>
+    <DefineConstants>TRACE</DefineConstants>
+    <ErrorReport>prompt</ErrorReport>
+    <WarningLevel>4</WarningLevel>
+    <CodeAnalysisRuleSet>AllRules.ruleset</CodeAnalysisRuleSet>
+    <Prefer32Bit>false</Prefer32Bit>
+  </PropertyGroup>
+  <ItemGroup>
+    <Reference Include="Alachisoft.Integrations.EntityFramework.CachingProvider">
+      <HintPath>..\..\..\..\..\..\integrations\MSEntityFramework\6.2\Alachisoft.Integrations.EntityFramework.CachingProvider.dll</HintPath>
```
Comment (Sarah-Hameed) [inline dotnet/EntityFramework/objectcaching/EFObjectCaching/EFObjectCaching.csproj:71]: proper reference?

File: dotnet/NCacheLINQ/NCacheLINQ/NCacheLINQ.csproj [hunk 30]
```diff
@@ -53,64 +53,76 @@
...
     </Reference>
     <Reference Include="log4net, Version=1.2.10.0, Culture=neutral, PublicKeyToken=1b44e1d426115821, processorArchitecture=MSIL">
-      <HintPath>..\packages\Alachisoft.NCache.SDK.5.0.2\lib\net40\log4net.dll</HintPath>
+      <HintPath>..\packages\Alachisoft.NCache.SDK.5.2.0\lib\net40\log4net.dll</HintPath>
+    </Reference>
+    <Reference Include="Newtonsoft.Json, Version=11.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed, processorArchitecture=MSIL">
+      <HintPath>..\packages\Newtonsoft.Json.11.0.2\lib\net40\Newtonsoft.Json.dll</HintPath>
     </Reference>
     <Reference Include="Oracle.DataAccess">
       <HintPath>..\..\..\..\Resources\build\64-bit\4.0\Oracle.DataAccess.dll</HintPath>
```
Comment (Sarah-Hameed) [inline dotnet/NCacheLINQ/NCacheLINQ/NCacheLINQ.csproj:108]: if folder dont exists?

File: dotnet/SignalRChat/SignalRChat/SignalRChat.csproj [hunk 31]
```diff
@@ -44,11 +44,56 @@
     <WarningLevel>4</WarningLevel>
   </PropertyGroup>
   <ItemGroup>
-    <Reference Include="Alachisoft.NCache.SignalR">
-      <HintPath>..\..\..\..\integrations\SignalR\Alachisoft.NCache.SignalR.dll</HintPath>
```
Comment (Sarah-Hameed) [inline dotnet/SignalRChat/SignalRChat/SignalRChat.csproj:48]: wrong reference, use nuget

File: dotnet/ContinuousQuery/ContinuousQuery/ContinuousQuery.csproj [hunk 32]
```diff
@@ -63,57 +63,9 @@
     <CodeAnalysisRuleDirectories>;C:\Program Files (x86)\Microsoft Visual Studio 10.0\Team Tools\Static Analysis Tools\FxCop\\Rules</CodeAnalysisRuleDirectories>
   </PropertyGroup>
   <ItemGroup>
-    <Reference Include="Alachisoft.NCache.Cache, Version=5.0.0.0, Culture=neutral, PublicKeyToken=cff5926ed6a53769, processorArchitecture=MSIL">
```
Comment (Sarah-Hameed) [inline dotnet/ContinuousQuery/ContinuousQuery/ContinuousQuery.csproj:66]: Add reference to SDK
